.github/workflows/ci.yml

```diff
@@ -1,9 +1,8 @@
 name: Continuous Integration
-on: 
+on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
```

Comment:
I would keep this. Only pull requests agianst master probably need checks.

Reply:
I had to drop this in https://github.com/facebook/taste-tester/pull/140 as otherwise it wouldn't run tests at all, and figured it'd be better to keep them the same for consistency.

Reply:
huh? I use tihs currently and it works fine. Show me?

Reply:
It's only a problem the first time you make the PR. Because you're making a PR from a local branch (and not a fork), it won't fire unless you allow non-master branches. In the first iteration of that PR (with `branches: [master]`) I only had CircleCI entries, and the action never fired.

Reply:
That's strange. that should be the branch you're making the pullrequest TO, not FROM.